README.md

```diff
@@ -33,6 +33,21 @@ sudo cp ./stine /usr/bin/stine
 Create a file called `.stine.yaml` (either inside your home folder or the folder containing the program) (e.g. using `nano ~/.stine.yaml`) and edit it to fit your needs.
 See [config](#config) section for examples.
 
+### macOS
+Download and unpack the latest release:
+```bash
```

Comment:
please add an empty line between the heading, text and code block
(_I know that I also broke the markdown rules - do as I say, not as I do_ :sweat_smile: )